content/docs/quickstart/kotlin.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: quickstart
+title: Kotlin Quick Start
+short: Kotlin
+description: This guide gets you started with gRPC in Kotlin with a simple working example.
+---
+
+<div id="toc"></div>
```

Comment:
This is no longer necessary

Reply:
Right, thanks!

I was waiting for #119 to be merged, before cleaning this up, but thanks for the reminder. I'll make other page adjustments, as necessary, now that #119 has been merged.